app/views/curation_concerns/file_sets/media_display/_default.html.erb

```diff
@@ -1,3 +1,6 @@
 <div class="no-preview">
   <%= t('sufia.works.show.no_preview') %>
+  <% if CurationConcerns.config.display_media_download_link %>
+    <p/><%= link_to t('sufia.file_set.show.download'), main_app.download_path(file_set), target: "_new", title: "Download the document" %>
```

Comment:
We've went and "button-ized" the link in CC, but this hasn't been completed just yet. I think that the `<p>` is fine until we've finalized and merged the branch in CC.

https://github.com/projecthydra/curation_concerns/blob/1adce97ca575fc2cf5ced80744a8efabe234d4e2/app/views/curation_concerns/file_sets/media_display/_default.html.erb#L8